project-1_test/main.py

```diff
@@ -0,0 +1,144 @@
+import inspect
+import logging
+import types
+
+import gspread
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+from selenium.webdriver.chrome.service import Service as ChromeService
+from webdriver_manager.chrome import ChromeDriverManager
+from selenium.common.exceptions import NoSuchElementException
+import os, json
+from dotenv import load_dotenv
+load_dotenv()
+KEY = os.getenv("PROJECT_1202")
+option = webdriver.ChromeOptions()
+option.add_argument('--headless')
+driver = webdriver.Chrome(service=ChromeService(ChromeDriverManager().install()), options=option)
+link = "https://www.lusha.com/company-search/accounting/10/canada/193/page/2/"
+driver.get(url=link)
+
+gc = gspread.service_account("/sadique_timileyin/project-1_test/my-project.json")
+
+
+def open_spreadsheet(sheet_name):
+    """
+    This opens a spreadsheet already created on Google sheet
+    """
+    if isinstance(sheet_name, str):
+        sh = gc.open(sheet_name)
+        logging.info("spreadsheet opened")
+        return sh
+    else:
+        raise ValueError("The sheet name expected a string object")
+
+
+def create_spreadsheet(sheet_name):
+    """
+    This creates a new spreadsheet file in google sheet
+    """
+    if isinstance(sheet_name, str):
```

Comment:
`str` is a keyword in python, rename this variable

Reply:
That is the second argument for the isinstance function.

Reply:
I see, my bad